pkg/validation/options.go

```diff
@@ -233,7 +233,14 @@ func parseProviderInfo(o *options.Options, msgs []string) []string {
 	p.ValidateURL, msgs = parseURL(o.ValidateURL, "validate", msgs)
 	p.ProtectedResource, msgs = parseURL(o.ProtectedResource, "resource", msgs)
 
-	o.SetProvider(providers.New(o.ProviderType, p))
+	provider := providers.New(o.ProviderType, p)
+	if provider == nil {
+		msgs = append(msgs, fmt.Sprintf("invalid setting: provider '%s' is not available", o.ProviderType))
+		return msgs
+	}
+	logger.Printf("configuring provider '%s'", o.ProviderType)
```

Comment:
We already log this in the core of the project so we don't need to log it here as well

https://github.com/oauth2-proxy/oauth2-proxy/blob/6254ed24ea319302b8cbdd8aaa16f6495c0b4c11/oauthproxy.go#L140

Reply:
Ok, wil remove

Reply:
Fixed.